README.md

```diff
@@ -69,14 +69,14 @@ Voterunner is a [node.js](http://nodejs.org/) app using
 [PostgreSQL](http://www.postgresql.org/) as a database so the following
 lines will bring it up:
 
-    $ git clone https://git.gitorious.org/electomat/voterunner.git
+    $ git clone https://github.com/xi/voterunner
     $ cd voterunner
     $ npm install
-    $ export PORT=5000
-    $ export DATABASE_URL='postgresql://user:password@host/database'
-    $ node app.js
-        info  - socket.io started
-        info  - Listening on 5000
+    $ pip install -r watch_requirements.txt
+    $ bin/initialize_db.sh
+    $ bin/watch.py
```

Comment:
IMHO watching should be optional. Could you please restore something like the original text (now including `initialize_db.sh`) and add an extra section about live reload?